### PR TITLE
Setup Android App With More Accurate settings

### PR DESCRIPTION
**Pull request by PureWeen**
### Description of Change

- Rework the android appium app to correctly setup package ids
- Rework reset to use terminate/launch which appears to be more reliable on Android

### CODE REVIEW

**Review by jsuarezruiz — DISMISSED** [review]

**Comment by jsuarezruiz** [comment]
LGTM! Waiting for build results.

File: src/TestUtils/src/UITest.Appium/HelperExtensions.cs [hunk 1]
```diff
@@ -541,6 +550,41 @@ public static void Back(this IApp app)
...
+			if (appId is not null)
+			{
+				return appId;
+			}
+
+			throw new InvalidOperationException("AppId not found");
+		}
+
+		/// <summary>
+		/// Retrive the target device this test is running against
```
Comment (jsuarezruiz) [inline src/TestUtils/src/UITest.Appium/HelperExtensions.cs:574]: Retrive -> Retrieve

Comment (PureWeen) [reply src/TestUtils/src/UITest.Appium/HelperExtensions.cs]: Done!

**Review by jsuarezruiz — APPROVED** [review]

**Comment by PureWeen** [comment]
Failing tests are unrelated

**Pull request merged**